code/WorkInProgress/cryospawn.dm

```diff
@@ -241,7 +248,11 @@
 			return 0
 		if (add_person_to_queue(user, null))
 			stored_mobs[user] = null
-			stored_mobs -= user
+			stored_mobs -= user.real_name
```

Comment:
This doesn't seem right. Are you sure?

Reply:
This is indeed not correct. Not exactly sure why I wrote it like that. Fixed on remote branch now.